platform/android/file_access_android.h

```diff
@@ -76,6 +76,8 @@ class FileAccessAndroid : public FileAccess {
 	virtual bool file_exists(const String &p_path) override; // return true if a file exists
 
 	virtual uint64_t _get_modified_time(const String &p_file) override { return 0; }
+	virtual uint64_t _get_access_time(const String &p_file) override { return 0; }
+	virtual uint64_t _get_size(const String &p_file) override { return 0; }
```

Comment:
This can be implemented using [`AAsset_getLength()`](https://developer.android.com/ndk/reference/group/asset#aasset_getlength) and [`AAsset_getLength64()`](https://developer.android.com/ndk/reference/group/asset#aasset_getlength64)